lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -110,6 +111,22 @@ llvm::Error setTensorType(const caffe2::QTensorProto &in, Tensor *T) {
     RETURN_ERR("Only uint8 and int32 qtensors are supported");
   }
 }
+
+Storage *createPlaceholderFromWeightTensor(
+    PlaceholderBindings *bindings, Function &f, Tensor &weights,
+    const std::string &name, bool allowMove, bool isQuantized,
```

Comment:
You should be able to remove `isQuantized` and instead just use the helper `isQuantizedElemKind(type)`